ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -144,115 +144,116 @@
   {{/unless}}
 
   {{! Signing Algorithms }}
-  <Rose::Form::Fieldset>
-    <:title>{{t 'form.signing_algorithms.label'}}</:title>
-    <:description>{{t 'form.signing_algorithms.help'}}</:description>
-    <:body>
-      {{#if @model.errors.signing_algorithms}}
-        <Rose::Form::Errors as |errors|>
-          {{#each @model.errors.signing_algorithms as |error|}}
-            <errors.message>{{error.message}}</errors.message>
-          {{/each}}
-        </Rose::Form::Errors>
-      {{/if}}
-      <Rose::Table @style='condensed' as |table|>
-        <table.header as |header|>
-          <header.row @hidden={{true}} as |row|>
-            <row.headerCell>{{t 'titles.value'}}</row.headerCell>
-            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model.signing_algorithms as |algorithm i|}}
-            <body.row as |row|>
-              <row.cell>
 
+  <Hds::Form::Fieldset as |F|>
```

Comment:
This should be converted into a new sub component for the `list-wrapper` to support selects

Reply:
Yeah, you can open a new PR for it and then use it in this one